[2022-04-28 06:06:44,810] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.rm_files_task scheduled__2004-01-01T00:00:00+00:00 [queued]>
[2022-04-28 06:06:44,864] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.rm_files_task scheduled__2004-01-01T00:00:00+00:00 [queued]>
[2022-04-28 06:06:44,866] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-28 06:06:44,869] {taskinstance.py:1250} INFO - Starting attempt 1 of 2
[2022-04-28 06:06:44,870] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-28 06:06:44,936] {taskinstance.py:1270} INFO - Executing <Task(BashOperator): rm_files_task> on 2004-01-01 00:00:00+00:00
[2022-04-28 06:06:44,946] {standard_task_runner.py:52} INFO - Started process 2105 to run task
[2022-04-28 06:06:44,960] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcp_dag', 'rm_files_task', 'scheduled__2004-01-01T00:00:00+00:00', '--job-id', '551', '--raw', '--subdir', 'DAGS_FOLDER/ingest_to_gcs_dag.py', '--cfg-path', '/tmp/tmp988888ac', '--error-file', '/tmp/tmpwgxn27zi']
[2022-04-28 06:06:44,964] {standard_task_runner.py:80} INFO - Job 551: Subtask rm_files_task
[2022-04-28 06:06:45,469] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcp_dag.rm_files_task scheduled__2004-01-01T00:00:00+00:00 [running]> on host d4721d223bdd
[2022-04-28 06:06:45,950] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-28 06:06:46,167] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcp_dag
AIRFLOW_CTX_TASK_ID=rm_files_task
AIRFLOW_CTX_EXECUTION_DATE=2004-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2004-01-01T00:00:00+00:00
[2022-04-28 06:06:46,174] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-28 06:06:46,178] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'rm /opt/***/laucnty04.xlsx /opt/***/us_employment_04.parquet']
[2022-04-28 06:06:46,201] {subprocess.py:85} INFO - Output:
[2022-04-28 06:06:46,299] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-28 06:06:46,519] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcp_dag, task_id=rm_files_task, execution_date=20040101T000000, start_date=20220428T060644, end_date=20220428T060646
[2022-04-28 06:06:46,585] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-28 06:06:46,813] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-04-29 02:36:52,629] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.rm_files_task scheduled__2004-01-01T00:00:00+00:00 [queued]>
[2022-04-29 02:36:52,684] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.rm_files_task scheduled__2004-01-01T00:00:00+00:00 [queued]>
[2022-04-29 02:36:52,688] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-29 02:36:52,692] {taskinstance.py:1250} INFO - Starting attempt 1 of 2
[2022-04-29 02:36:52,694] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-29 02:36:52,752] {taskinstance.py:1270} INFO - Executing <Task(BashOperator): rm_files_task> on 2004-01-01 00:00:00+00:00
[2022-04-29 02:36:52,763] {standard_task_runner.py:52} INFO - Started process 5608 to run task
[2022-04-29 02:36:52,774] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcp_dag', 'rm_files_task', 'scheduled__2004-01-01T00:00:00+00:00', '--job-id', '722', '--raw', '--subdir', 'DAGS_FOLDER/ingest_to_gcs_dag.py', '--cfg-path', '/tmp/tmp6p7fmizf', '--error-file', '/tmp/tmpczf16ub1']
[2022-04-29 02:36:52,793] {standard_task_runner.py:80} INFO - Job 722: Subtask rm_files_task
[2022-04-29 02:36:53,183] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcp_dag.rm_files_task scheduled__2004-01-01T00:00:00+00:00 [running]> on host 6fd0ba274d58
[2022-04-29 02:36:53,451] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-29 02:36:53,663] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcp_dag
AIRFLOW_CTX_TASK_ID=rm_files_task
AIRFLOW_CTX_EXECUTION_DATE=2004-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2004-01-01T00:00:00+00:00
[2022-04-29 02:36:53,668] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-29 02:36:53,673] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'rm /opt/***/laucnty04.xlsx /opt/***/us_employment_2004.parquet']
[2022-04-29 02:36:53,697] {subprocess.py:85} INFO - Output:
[2022-04-29 02:36:53,773] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-29 02:36:54,065] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcp_dag, task_id=rm_files_task, execution_date=20040101T000000, start_date=20220429T023652, end_date=20220429T023654
[2022-04-29 02:36:54,153] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-29 02:36:54,398] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
